src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/profileSettings/profileProperties/profilePropertyEditor/index.jsx

```diff
@@ -89,19 +103,32 @@ class ProfilePropertyEditor extends Component {
             if (props.propertyLocalization["PropertyName"] === "" || props.propertyLocalization["PropertyName"] === undefined) {
                 state.error["localeName"] = true;
             }
-            else if (props.propertyLocalization["PropertyName"] !== "" && props.propertyLocalization["PropertyName"] !== undefined) {
+            else {
                 state.error["localeName"] = false;
             }
         }
 
         this.setState({
             profileProperty: Object.assign({}, props.profileProperty),
             propertyLocalization: Object.assign({}, props.propertyLocalization),
-            triedToSubmit: false,
+            triedToSubmit: state.triedToSubmit,
             error: state.error
         });
     }
 
+    isValidName(name) {
+        const validatePropertyName = /^[a-zA-Z0-9]+$/g;
```

Comment:
Please check how the app behaves if the property started with a digit. I wouldn't be surprised if the first character would have to be a letter.

Reply:
@tpluscode, I tested with properties starting as numerics and works fine. Component identifiers are concatenated with property names making the variables never start with the name of the property.